Userland/Libraries/LibGUI/Shortcut.cpp

```diff
@@ -42,4 +42,15 @@ String Shortcut::to_string() const
     return builder.to_string();
 }
 
+bool Shortcut::looks_like_text_entry() const
+{
+    if (m_type != Type::Keyboard)
+        return false;
+    if (m_modifiers & Mod_Alt || m_modifiers & Mod_Ctrl || m_modifiers & Mod_Super)
+        return false;
+
+    // FIXME: Support other letters besides a-z, 0-9?
+    return ((m_keyboard_key >= Key_A && m_keyboard_key <= Key_Z) || (m_keyboard_key >= Key_0 && m_keyboard_key <= Key_9));
```

Comment:
This should maybe include symbols like: `" < / . { + ~` etc.
But I guess this can be a good start and I don't know if there are any shortcuts that actually use those

Reply:
Looks like Key_Space to Key_Backtick should cover all of them? https://github.com/SerenityOS/serenity/blob/9ad6031bca24d1400619c384b89f437a03a81813/Kernel/API/KeyCode.h#L48

Reply:
Yeah looks like a good starting set!

Reply:
Latest push uses this range (Space to Backtick). Thanks for the feedback on that :)